src/dune_rules/coq/coq_config.ml

```diff
@@ -154,8 +154,7 @@ type t =
 let impl_config bin =
   let* _ = Build_system.build_file bin in
   Memo.of_reproducible_fiber
-  @@ Process.run_capture_lines ~display:!Clflags.display Process.Return bin
-       [ "--config" ]
+  @@ Process.run_capture_lines ~display:Quiet Process.Return bin [ "--config" ]
```

Comment:
Here we query the configuration of Coq. This is internal and does not need to be reported to the user.